how-to.html

```diff
@@ -0,0 +1,80 @@
+---
+layout: two-col
+---
+
+<div class="margin-trailer">
+  <h2>What to Share</h2>
+  <p>
+    Stories submitted to The Second Page should be largely true, and involve your own experiences.
```

Comment:
We aired a couple fiction and/or experimental stories in the first season, so we could add a sentence about having some flexibility around that, but it't not critical.

Reply:
Updated to:
```
    Stories submitted to The Second Page should be largely true, and involve your own experiences (though some artistic liberty
    can be appreciated).
```